### PR TITLE
Preact to react

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are switching the preact wails front-end app to react

### CODE REVIEW

File: wails-frontend/src/events/events.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export type promptResult = {
   Success: boolean;
-  Data: any;
+  Data: [];
```
Comment (peterjah) [inline wails-frontend/src/events/events.tsx:3]: why do you assume Data is an array ? it is not the case, for now its a string. please let "any" for now. if the linter complais you can disable it for this file!

**Review by peterjah — CHANGES_REQUESTED** [review]

File: wails-frontend/src/events/events.tsx (hunk 1 above)
Comment (maxime-carabina) [reply wails-frontend/src/events/events.tsx:3]: Ok no problem !

Comment (maxime-carabina) [reply wails-frontend/src/events/events.tsx]: Done @peterjah

**Review by peterjah — APPROVED** [review]

**Comment by Thykof** [comment]
> what were the motivations?

ui-kit was not importable into preact wail frontend

**Pull request merged**